src/database/index.tsx

```diff
@@ -368,28 +373,52 @@ export const saveFacility = async (facility: any): Promise<void> => {
       // Ensures we don't store any attributres that our model does not
       // know about.
       facility.modelInputs = pick(facility.modelInputs, persistedKeys);
+      facility.modelInputs.updatedAt = currrentTimestamp();
+      // TODO: For now, this assumes we're always entering data as of "today"
```

Comment:
Given that the text box that will allow users to input this doesn't exist yet, I am 100% okay deferring. Thanks for the thoughtfulness!